src/h5web/visualizations/nexus/hooks.ts

```diff
@@ -65,7 +66,12 @@ export function useNxData(group: HDF5Group, metadata: HDF5Metadata): NxData {
 
   const axesNames = getNxAxisNames(group);
 
-  const axisMapping = axesNames.map((name) => {
+  const silxStyle = getAttributeValue(group, 'SILX_style');
+  const scaleStyles: SilxStyle | undefined =
+    typeof silxStyle === 'string' ? JSON.parse(silxStyle) : undefined;
```

Comment:
You may want to add ` && silxStyle` to the condition, as `JSON.parse("")` throws.

Reply:
Ha :fishing_pole_and_fish: ! 

Thanks, not used to `JSON.parse`